packages/trader/src/Modules/Trading/Components/Form/TradeParams/Multiplier/cancel-deal.tsx

```diff
@@ -54,7 +54,7 @@ const CancelDeal = observer(() => {
                         message={
                             <PopoverMessageCheckbox
                                 defaultChecked={is_do_not_show_selected}
-                                checkboxLabel={localize("Don't show this again")}
+                                checkboxLabel={<Localize i18n_default_text="Don't show this again" />}
                                 message={localize(
```

Comment:
`localize()`? Or you change this line in another contract?
Also if `localize()` is removed everywhere, that it can be also deleted from imports)

Reply:
@kate-deriv this message needs type change for PopoverMessageCheckbox, i'm adding localizes for children that don't need the type change now )